src/main/java/org/apache/sling/servlets/annotations/SlingServletResourceTypes.java

```diff
@@ -43,18 +43,26 @@
      * @return the resource type(s)
      */
     String[] resourceTypes();
-    
+
+    /**
+     * The resource super type from which the servlet inherits (value is "sling.servlet.resourceSuperType").
+     *
+     * @return the resource super type
```

Comment:
Can you add a hint from which version of the servlet resolver bundle this is supported and what happens if that is not set (i.e. default handling)

Reply:
Yes, I was just doing this for the "Servlets and Scripts" site page and was thinking that it would be worth adding this info here too. :)